packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+
+import ConditionalSuspense from './ConditionalSuspense';
+import useIsBrowser from './useIsBrowser';
+import useIsIntersecting from './useIsIntersecting';
+
+import type { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+interface FallbackProps extends Omit<MuxPlayerProps, 'playerSoftwareVersion' | 'playerSoftwareName'> {
+  onIntersection?: () => void;
+}
+const Fallback = (props: FallbackProps) => {
+  const { style, onIntersection, ...rest } = props;
+
+  const intersectionRef = React.useRef<HTMLElement>(null);
+  const isIntersecting = useIsIntersecting(intersectionRef);
+
+  useEffect(() => {
+    if (isIntersecting && onIntersection) {
+      onIntersection();
+    }
+  }, [isIntersecting, onIntersection]);
+
+  return (
+    <>
+      {/* TODO: can we add mux-player to JSX.IntrinsicElements */}
+      {/* @ts-ignore */}
+      <mux-player
+        ref={intersectionRef}
+        data-mux-player-react-placeholder
+        style={{
+          // default mux-player styling
+          aspectRatio: '16/9',
+          display: 'block',
+          backgroundColor: '#000',
+          width: '100%',
+          // some styles specific to this placeholder
+          position: 'relative',
+          backgroundImage: props.placeholder ? `url(${props.placeholder})` : undefined,
+          backgroundRepeat: 'no-repeat',
+          backgroundSize: 'contain',
+          // user-implemented styles
+          ...style,
+        }}
+        // we pass the rest of the props to mux-player too.
+        // There is a possibility that mux-player imports before Suspense has had the opportunity
+        // to replace this placeholder with MuxPlayerReact.
+        // In that case, we want this placeholder to look as much like the incoming player as possible.
+        {...rest}
+      >
+        {/* Loading Spinner SVG */}
+        <svg
+          aria-hidden="true"
+          style={{
+            position: 'absolute',
+            top: '50%',
+            left: '50%',
+            width: '100px',
+            height: '100px',
+            transform: 'translate(-50%,-50%)',
+          }}
+          viewBox="0 0 100 100"
+          fill="transparent"
+        >
+          <path d="M73,50c0-12.7-10.3-23-23-23S27,37.3,27,50 M30.9,50c0-10.5,8.5-19.1,19.1-19.1S69.1,39.5,69.1,50">
+            {/* after half a second, animate fill to appear */}
+            <animate attributeName="fill" dur="0.1s" from="transparent" to="#fff" begin="0.5s" fill="freeze" />
+            {/* spin! */}
+            <animateTransform
+              attributeName="transform"
+              attributeType="XML"
+              type="rotate"
+              dur="1s"
+              from="0 50 50"
+              to="360 50 50"
+              repeatCount="indefinite"
+            />
+          </path>
+        </svg>
```

Comment:
I think we shouldn't show a loading spinner while loading.
it looks a bit distracting to me and the blurry image already gives an indication that something is loading there.
it it has multiple players on a page it's gonna look too noisy with all the spinners.

examples: 
https://player.vimeo.com/video/638369396
https://fast.wistia.net/embed/iframe/oifkgmxnkb?twitter=true

Reply:
Yeah, loading spinner often makes things seem slower. Potentially, worth showing the loading spinner if it's taking more than a couple of seconds to load.

Reply:
Loading spinner pops in after half a second, here. Though yeah I'm considering removing it altogether. Next image works fine without a spinner. I think I'll remove it unless anyone's got strong opinions otherwise.

Reply:
half a second is too short. Should be like 2 seconds minimum. I'm good with just removing it.

Reply:
it gon